src/LiveSplit.WorldRecord/UI/Components/WorldRecordComponent.cs

```diff
@@ -164,10 +164,11 @@ private void ShowWorldRecord(LayoutMode mode)
                     timingMethod = game.Ruleset.DefaultTimingMethod.ToLiveSplitTimingMethod();
                 }
 
-                LocalTimeFormatter.Accuracy = game.Ruleset.ShowMilliseconds ? TimeAccuracy.Milliseconds : TimeAccuracy.Seconds;
+                LocalTimeFormatter.Accuracy = CheckPrecisionMillis(timeInfo) ? TimeAccuracy.Milliseconds : TimeAccuracy.Seconds;
+                WRTimeFormatter.Accuracy = CheckPrecisionMillis(timeInfo) ? TimeAccuracy.Milliseconds : TimeAccuracy.Seconds;
```

Comment:
After my change https://github.com/LiveSplit/LiveSplit.WorldRecord/commit/0ea93d1149e24002b5177bc4d1d455afd6d16e88, I think it would make sense to disable automatic precision if a precision (seconds or milliseconds) is explicitly chosen by the user?

Reply:
By the way, please merge `master` into your branch so that you have the latest code